test/Driver/android-sdk.swift

```diff
@@ -0,0 +1,4 @@
+// RUN: %swiftc_driver_plain -frontend -target x86_64-unknown-linux-android -sdk %S/Inputs/android.sdk %s -typecheck
```

Comment:
Instead of `%swiftc_driver_plain`, can you use `%swift`? This will stick something in the global module cache instead of the test-specific one.

Reply:
Eek, thanks for that info.  I’ll adjust that.